media/livestream/livestream_test.go

```diff
@@ -60,6 +64,12 @@ func TestLiveStream(t *testing.T) {
 
 	testChannelEvents(t, outputURI)
 	t.Logf("\ntestChannelEvents() completed\n")
+
+	testAssets(t, assetURI)
```

Comment:
These tests should probably be in their own test function (e.g. `TestAssets`), if they do not depend on the previous tests in this method. same for `testPools()`.

This will make them run faster, and ensure that the Asset tests still run if there are problems with other livestream tests.

Reply:
Done